UPDATING.md

```diff
@@ -31,6 +31,8 @@ assists people when migrating to a new version.
 
 ### Potential Downtime
 
+- [16756](https://github.com/apache/incubator-superset/pull/16756): The renaming of column `dbs.allow_csv_upload` fails for MySQL 8 users because there's a check constraint on it.
```

Comment:
```suggestion
- [16756](https://github.com/apache/incubator-superset/pull/16756): a change which renames the `dbs.allow_csv_upload` column to `dbs.allow_file_upload` via a (potentially locking) DDL operation.
```

Reply:
@exemplary-citizen it seems like @betodealmeida fix the MySQL 8 issue in the same migration and thus your original comment isn't really relevant.